### DISCUSSION TITLE
How to using Archive chat without Chat Render(only using new chat player)

### DISCUSSION THREAD

**Discussion by kkr881** [Zibbp/ganymede#70] [Q&A] [answered]
Hi! I'm using Ganymede well.

But I have a question.
I'm setting up Archive Chat when registering for Watched Channels, but I only need a new chat player, so I don't want to do Chat Render that requires a lot of performance and takes a long time.

In this case, how should I set it up? Please answer me!

**Comment by Zibbp** [comment, marked as answer]
Hi,
Currently there is no way to disable chat rendering while archiving. Ganymede is an archiving platform and the chat render will outlast the new chat player by many years as it's a just a video which is why it is the default.

I may allow the option to disable chat rendering in the future but right now it will stay as the default.

**Comment by kkr881** [comment]
Thanks. I understood what you were saying.

**Comment by Zibbp** [comment]
@kkr881 
Hi,
[Version 1.1.7](https://github.com/Zibbp/ganymede/releases/tag/v1.1.7) has been released which includes the ability to disable chat rendering for an archive while still downloading the chat.json. Hope this helps!

**Reply by bonny1992** [reply]
Hey @Zibbp it's me again :)

I tried disabling Chat Rendering because it's just too much time consuming (to the point that big past broadcasts will expire, especially if something fails in-between) so I thought of trying disabling it from the Watched Channel setting.

The issue is that even disabling it (new instance, only 1 channel) it will render it nonetheless!

Channel setting (I disabled the render when I first added the Watched Channel ofc):
![Channel setting](https://user-images.githubusercontent.com/1154368/213949687-013a4a60-9dc7-4aec-8181-8f836504e14f.jpg)

Queue item (just installed Ganymede, deleting the db):
![Queue item](https://user-images.githubusercontent.com/1154368/213949716-7b70d4b0-d4f3-4a63-8afc-60b965d8744d.jpg)

Chat render log:
![Chat render log](https://user-images.githubusercontent.com/1154368/213949725-96ab9660-e6aa-40ac-85fa-ca61041df299.jpg)

**Reply by Zibbp** [reply]
Hi, thanks for reporting the bug. It has been fixed in https://github.com/Zibbp/ganymede/commit/2c32b7b574ffa40c1979de893aad381af119b6d2. I'll update you here when a release is pushed out that includes this fix.

**Reply by bonny1992** [reply]
Hey @Zibbp any update on the release? :)
Can't wait for it!

**Reply by Zibbp** [reply]
The fix was released awhile ago, forgot to reply oops. Have you tried it out recently on the latest version?

**Comment by kkr881** [comment]
Oh! Thank you.
I think this option will be very useful and very helpful to me!

**Comment by bonny1992** [comment]
> The fix was released awhile ago, forgot to reply oops. Have you tried it out recently on the latest version?

To be fair no, I put everything on pause waiting (no problem of course).
I'll try it soon then!
